changelog.md

```diff
@@ -47,6 +47,9 @@
 * [Issue 114](https://github.com/jacob-carlborg/dstep/issues/114): Crash on recursive typedef.
 * [Issue 116](https://github.com/jacob-carlborg/dstep/issues/116): Option --space-after-function-name doesn't work with function pointer syntax.
 * [Issue 117](https://github.com/jacob-carlborg/dstep/issues/117): fatal error: 'limits.h' file not found.
+* [Issue 137](https://github.com/jacob-carlborg/dstep/issues/137): struct member expansion.
+* [Issue 138](https://github.com/jacob-carlborg/dstep/issues/138): Repeated declarations cause problems.
+* [Issue 140](https://github.com/jacob-carlborg/dstep/issues/140): On enums and scope.
```

Comment:
The fix for this issue actually introduced a new feature, which should be mentioned in the `
New/Changed Features` section.